lib/daru/dataframe.rb

```diff
@@ -705,6 +705,14 @@ def rolling_fillna(direction=:forward)
       dup.rolling_fillna!(direction)
     end
 
+    # Return unique rows by vector specified or all vectors
+    def unique(vecs=[])
```

Comment:
a) method should be named `uniq` (see Ruby's Array API: https://docs.ruby-lang.org/en/2.4.0/Array.html#method-i-uniq)
b) It probably should also support blocks, like `Array#uniq`